edx_username_changer/exceptions.py

```diff
@@ -0,0 +1,15 @@
+"""
+Execeptions for edx-username-changer plugin
+"""
+
+
+class UpdateFailedException(Exception):
+    def __init__(self, url, new_username):
+        self.url = url
+        self.new_username = new_username
+
+    def __str__(self):
+        return "Username update failed (username update attempted: {}, url: {})".format(
```

Comment:
I think we should we change this representation? e.g. 
```suggestion
        return "Username update failed for username: {}, url: {})".format(
```

Reply:
Updated.